gateway-service/src/main/java/org/zowe/apiml/gateway/config/ConnectionsConfig.java

```diff
@@ -125,7 +125,13 @@ public class ConnectionsConfig {
     @Value("${eureka.client.serviceUrl.defaultZone}")
     private String eurekaServerUrl;
 
-    @Value("${apiml.gateway.timeout:60}")
+    @Value("${apiml.httpclient.conn-pool.idleConnTimeoutSeconds:#{5}}")
```

Comment:
I guess we want to replace this old property with a new one, don't we?

Reply:
Unified under `apiml.connection`. However we should use only one single HttpConfig, the one defined in the apiml-common. It can be part of different PR.